owncloudApp/src/main/java/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -478,4 +481,9 @@ protected boolean isShowingJustFolders() {
         Bundle args = getArguments();
         return ((args != null) && args.getBoolean(ARG_JUST_FOLDERS, false));
     }
+
+    protected boolean isShowingOnlyAvailableOffline() {
```

Comment:
protected not needed. Android Studio warns you about that.

![Screenshot 2019-03-13 at 17 28 17](https://user-images.githubusercontent.com/15261363/54296547-6ce73680-45b5-11e9-8e8c-d75d9fb39e68.png)